Components/modal.html

```diff
@@ -0,0 +1,132 @@
+                    <a class="link" href="#">
+                        <li>Text-Utility</li>
+                    </a>
+                    <a class="link link-active" href="#">
+                        <li>Modal</li>
+                    </a>
+                </ul>
+        </div>
+        <div class="documentation">
+            <!-- <h1 class="documentation-component-main-heading">Modal</h1> -->
+            <div class="modal-container">
+                <div class="modal-card">
+                    <div class="inner-box" id="modal-card">
+                        <div class="card-front">
+                            <h2>LOGIN</h2>
+                            <form>
+                                <input type="email" class="form-input-box" placeholder="Enter your Email id" required>
+                                <input type="password" class="form-input-box" placeholder="Password">
+                                <button type="submit" class="submit-btn">
+                                    Submit
+                                </button>
+                                <div class="checkbox-remember-me">
+                                    <input type="checkbox"><span>Remember me</span>
+                                </div>
+                            </form>
+                            <button type="button" class="btn-new-here" onclick="openRegister()">I'm New Here</button>
+                            <a class="link-forgot-password" href="#">Forgot Password</a>
+                        </div>
+                        <div class="card-back">
+                            <h2>REGISTER</h2>
+                            <form>
+                                <input type="text" class="form-input-box" placeholder="Enter your Name">
+                                <input type="email" class="form-input-box" placeholder="Enter your Email id" required>
+                                <input type="password" class="form-input-box" placeholder="Password">
+                                <button type="submit" class="submit-btn">
+                                    Submit
+                                </button>
+                                <div class="checkbox-remember-me">
+                                    <input type="checkbox"><span>Remember me</span>
+                                </div>
+                            </form>
+                            <button type="button" class="btn-new-here" onclick="openLogin()">I've an Account</button>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- <div class="code-snippet"></div> -->
+        </div>
+    </div>
+    <script>
+        var modalCard = document.getElementById("modal-card");
+
+        function openRegister() {
+            modalCard.style.transform = "rotateY(-180deg)";
+        }
+
+        function openLogin() {
+            modalCard.style.transform = "rotateY(0deg)";
+        }
+    </script>
```

Comment:
JS code should not be in HTML

Reply:
Yes, I had to do it fast so took this shortcut but will change it later.